fbuapp/View Controllers/CreatePostViewController.m

```diff
@@ -334,6 +334,9 @@ -(void)didSelectCell: (NSIndexPath *)indexPath {
     self.eventCategory = indexPath.row;
 }
 
+- (void)didDeselectCell: (NSIndexPath *)indexPath{
+}
```

Comment:
should there be stuff in this function

Reply:
no but this method is necessary for the delegate